app/code/Magento/Msrp/Test/Unit/Pricing/MsrpPriceCalculatorTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Msrp\Test\Unit\Pricing;
+
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\Type as Type;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\GroupedProduct\Model\Product\Type\Grouped as GroupedType;
+use Magento\Msrp\Pricing\MsrpPriceCalculator;
+use Magento\MsrpGroupedProduct\Pricing\MsrpPriceCalculator as MsrpGroupedCalculator;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class MsrpPriceCalculatorTest extends TestCase
+{
+    /**
+     * Test getMrspPriceValue() with the data provider below
+     *
+     * @param array $msrpPriceCalculators
+     * @param Product $productMock
+     * @param float $expected
+     * @dataProvider getMsrpPriceValueDataProvider
+     */
+    public function testGetMsrpPriceValue($msrpPriceCalculators, $productMock, $expected)
+    {
+        $objectManager = new ObjectManager($this);
```

Comment:
Please move the object manager along with testable object initialization to the `setUp` method. No need to initialize them for each test case triggered.
Thus, it can be also used in further tests if necessary.